nifi-extension-bom/pom.xml

```diff
@@ -213,14 +213,14 @@
             </dependency>
             <dependency>
                 <groupId>org.mortbay.jasper</groupId>
-                <artifactId>apache-jsp</artifactId>
-                <version>10.1.43</version>
+                <artifactId>mortbay-apache-jsp</artifactId>
+                <version>10.1.44</version>
                 <scope>provided</scope>
             </dependency>
             <dependency>
                 <groupId>org.mortbay.jasper</groupId>
-                <artifactId>apache-el</artifactId>
-                <version>10.1.43</version>
+                <artifactId>mortbay-apache-el</artifactId>
+                <version>10.1.44</version>
```

Comment:
Since this is in `dependencyManagement`, changing the coordinates means the override will not apply to older versions. Do you know if all corresponding references use the new coordinates?

Reply:
Good point, pushed a commit to make sure we do not have anything coming with the legacy coordinates.